python/hyfetch/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               python 1.0
+PortGroup               github 1.0
+
+github.setup            hykilpikonna hyfetch 1.4.10
+revision                0
+categories-append       sysutils
+platforms               {darwin any}
+supported_archs         noarch
+license                 MIT
+maintainers             {@therealketo gmail.com:therealketo} openmaintainer
+
+description             Fast, highly customisable system info script with LGBTQ+ pride flags
+long_description        {*}${description}.
+
+checksums               rmd160  24e94b8d83d1e95174025e4da090b830b3ca94f7 \
+                        sha256  c8589ad16b2d3689a52c2dcb39462a5fdaa045ad29446b9803228f2f3ee0dec1 \
+                        size    267246
+
+depends_build-append    port:py${python.version}-typing_extensions
+
+post-destroot {
+    set hyfetch_path ${prefix}/Library/Frameworks/Python.framework/Versions/${python.branch}
+    ln -s ${hyfetch_path}/bin/${name} ${destroot}${prefix}/bin/${name}
+    ln -s ${hyfetch_path}/bin/neowofetch ${destroot}${prefix}/bin/neowofetch
```

Comment:
not sure why this is needed, why does the default installation using the `python` PG not work?

Reply:
The default installation works when setting `python.default_version` explicitly, will remove this to opt over that, but it's strange to me that such would be the default, yet it still needs to be explicitly specified in a Portfile.